### ISSUE TITLE
Activate / focus block with choices automatically

### ISSUE THREAD

**Issue by BITS-Editor** [bonartm/quizdown-js#22]
Is it possible to activate or focus the block with the choices when opening a quiz page automatically - or at least when going to the next page with the next question?

The demo https://bonartm.github.io/quizdown-live-editor/ seemingly (?) does not support keyboard navigation.

**Comment by bonartm** [comment]
I can navigate through the page with `Tab` (next focus) `Shift-Tab` (previous focus)  and `space` (select item, click button). When clicking the "next" button with the keyboad I'm still inside the quiz and I can go to the choices section by pressing `Shift-Tab`. 

With the demo, because of the text editor, there can be problems because of "tab trapping". It's a known issue but can be solved by pressing `Str+M` when inside the text editor (https://github.com/microsoft/monaco-editor/wiki/Monaco-Editor-Accessibility-Guide#tab-trapping).

**Comment by BITS-Editor** [comment]
> I can navigate through the page with `Tab` (next focus) `Shift+Tab` (previous focus) and `space` (select item, click button). When clicking the "next" button with the keyboad I'm still inside the quiz and I can go back to the choices section by pressing `Shift+Tab` repeatedly.

True, but navigating by `Shift+Tab` backwards on the subsequent pages of the quiz to the question block is a bit annoying. I would be grateful if you could check a remedy!
